docs/create-compressed-backup.md

```diff
@@ -1,79 +1,136 @@
 # Create a compressed backup
 
-Percona XtraBackup supports compressed backups. A local or streaming
-backup can be compressed or decompressed with *xbstream*.
+Percona XtraBackup supports compressed backups. A local or streaming backup can be compressed or decompressed with *xbstream*.
 
-!!! note
+To make a compressed backup, use the `--compress` option along with the `--backup` and `--target-dir` options. 
 
-    Starting with Percona XtraBackup 8.0.31-24 using qpress/QuickLZ to compress backups is deprecated and may be removed in future versions. We recommend using either `LZ4` or Zstandard (`ZSTD`) compression algorithms.
+## Version updates
 
-To make a compressed backup, use the `--compress` option along
-with the `--backup` and `--target-dir` options. 
+??? note "Starting with Percona XtraBackup 8.0.34-29"
```

Comment:
What does `???` means here?

Reply:
This is the markdown syntax we use to add tabs.